### PR TITLE
First iteration of a command line utility to seed vendor data

### PR DESCRIPTION
**Pull request by aaron-collier**
This needs some cleanup, but can at this time seed the database with vendor records

Screenshot 2023-05-04 at 10 11 51 AM

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -133,6 +133,13 @@ module and then run the following steps:
  1. From the root directory, run `poetry run alembic revision --autogenerate -m "{short message describing change}"` (**NOTE:** not all changes to the model are detected, see this [note](https://alembic.sqlalchemy.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect) in the documentation)
  1. After the migration script is created, run `poetry run alembic upgrade head` to apply your latest changes to the database.
 
+#### Seeding the database
```
Comment (mjgiarlo) [inline README.md:136]: Maybe this header can be removed since there's no text beneath it?

File: bin/seed_vendors [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Get vendor records from FOLIO Organizations")
+    parser.add_argument("--log", default="seed_vendors.log", help="A file path to write log messages")
+    opts = parser.parse_args()
+
+    logging.basicConfig(filename=opts.log, format="%(asctime)s - %(levelname)s - %(message)s", level=logging.INFO)
+
+    # Ignore broken pipe errors when running from the command line.
+    # This allows: bin/get penn penn_babylonian | head -10
```
Comment (justinlittman) [inline bin/seed_vendors:105]: Cruft

File: libsys_airflow/plugins/vendor/models.py [hunk 3]
```diff
@@ -26,8 +26,8 @@ class VendorInterface(Model):
     vendor_id = Column(Integer, ForeignKey('vendors.id'))
     display_name = Column(String(50), unique=False, nullable=False)
     folio_interface_uuid = Column(String(36), unique=True, nullable=False)
-    folio_data_import_profile_uuid = Column(String(36), unique=True, nullable=False)
-    folio_data_import_processing_name = Column(String(50), unique=False, nullable=False)
+    folio_data_import_profile_uuid = Column(String(36), unique=True, nullable=True)
+    folio_data_import_processing_name = Column(String(50), unique=False, nullable=True)
```
Comment (justinlittman) [inline libsys_airflow/plugins/vendor/models.py:30]: We don't actually need the name to invoke import.

**Review by justinlittman — APPROVED** [review]
Approve pending consideration of comments.

File: libsys_airflow/plugins/vendor/models.py (hunk 3 above)
Comment (aaron-collier) [reply libsys_airflow/plugins/vendor/models.py:30]: Right, we can capture it, but it's not needed so that's why I removed the `nullable=False` :-)

**Pull request merged**